packages/node/src/indexer/api.service.ts

```diff
@@ -160,11 +160,24 @@ export class ApiService
   }
 
   async init(): Promise<ApiService> {
+    return this.load();
+  }
+
+  async reloadChainTypes(chainTypes: unknown): Promise<void> {
```

Comment:
Do we have to recreate all the api instances? This seems quite inefficient, creating new polkadot api instances is slow.